src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallServiceTest.kt

```diff
@@ -119,19 +119,24 @@ class RecallServiceTest {
     assessedByUserId = fullyPopulatedUpdateRecallRequest.assessedByUserId!!.value,
     bookedByUserId = fullyPopulatedUpdateRecallRequest.bookedByUserId!!.value,
     dossierCreatedByUserId = fullyPopulatedUpdateRecallRequest.dossierCreatedByUserId!!.value,
-    dossierTargetDate = LocalDate.of(2021, 10, 5),
+    dossierTargetDate = dossierTargetDate(fullyPopulatedUpdateRecallRequest),
     lastKnownAddressOption = fullyPopulatedUpdateRecallRequest.lastKnownAddressOption,
     arrestIssues = fullyPopulatedUpdateRecallRequest.arrestIssues,
     arrestIssuesDetail = fullyPopulatedUpdateRecallRequest.arrestIssuesDetail,
     warrantReferenceNumber = fullyPopulatedUpdateRecallRequest.warrantReferenceNumber,
   )
 
+  private fun dossierTargetDate(fullyPopulatedUpdateRecallRequest: UpdateRecallRequest) =
+    if (fullyPopulatedUpdateRecallRequest.inCustodyAtAssessment ?: fullyPopulatedUpdateRecallRequest.inCustodyAtBooking!!) {
+      LocalDate.of(2021, 10, 5)
```

Comment:
Two literal uses -> one named val please, given the uses have to match and are in different methods?